doc/development/developing-sql-dialect/implementing_additional_dialect_specific_behavior.md

```diff
@@ -235,11 +235,11 @@ That way the Exasol database can directly run `IMPORT` on the remote data source
 
 ###  Pre-Requisites for Using IMPORT
 
-If you plan to use `IMPORT`, bare in mind that remote data source must **offer a JDBC driver** (always the case if you write a dialect for the JDBC adapter instead of an adapter from scratch).
+If you plan to use `IMPORT`, bear in mind that remote data source must **offer a JDBC driver** (always the case if you write a dialect for the JDBC adapter instead of an adapter from scratch).
 
 ### Overloading Rewriting
 
-The `AbstractSqlDialect` has a base implementation for query rewriting that should work with most databases that support JDBC. 
+The `AbstractSqlDialect` has a base implementation for the query rewriting that should work with most databases that support JDBC. 
```

Comment:
"the" is not necessary here.

Reply:
My IDE's spell checker was complaining about it